### ISSUE TITLE
Support inline constraint names in CREATE TABLE statements

### ISSUE THREAD

**Issue by MirkoTeran** [jOOQ/jOOQ#8386]
### Use case:

I would like to translate 100+ of DDL queries written for Postgres to other databases. Currently it doesn't support named inlined constraints in create table queries.

Example:

```sql
CREATE TABLE workspace_account(
  workspace_id integer NOT NULL CONSTRAINT workspace_account_fk_workspace REFERENCES workspace(id) ON DELETE NO ACTION, 
  account_id integer NOT NULL CONSTRAINT workspace_account_fk_account REFERENCES account(id) ON DELETE NO ACTION,
  CONSTRAINT workspace_account_pk PRIMARY KEY (workspace_id, account_id)
);
```

results in 

```
`Token ')' expected: [3:33] ...rkspace_account
(
  workspace_id integer NOT NULL [*]CONSTRAINT workspace_account_fk_workspace REFERENCES workspace(id) ON DELETE NO ...
```

### Possible solution you'd like to see:

Support for ... CONSTRAINT <name> syntax in DDL.

### Possible workarounds:

Removing the constraint names from source files and re-adding them in output by hand.

### Versions:

- jOOQ: 3.11.9 pro (also tested with https://www.jooq.org/translate/ - same results)
- Java: 11
- Database (include vendor): Postgres 9.6 - 11
- OS: Win10
- JDBC Driver (include name if inofficial driver):

**Comment by lukaseder** [comment]
Thank you very much for your request. Indeed, this particular syntax has already been on the list of things to support: #7518. I'll see that this will be included in jOOQ 3.12.

**Comment by lukaseder** [comment]
Your reported DDL now translates as expected on https://www.jooq.org/translate

![image](https://user-images.githubusercontent.com/734593/53801050-72ea7100-3f3e-11e9-932c-42a2ded03153.png)
